samila/functions.py

```diff
@@ -182,6 +182,7 @@ def plot_params_filter(
         size = g.size
     if projection is None:
         projection = g.projection
+    g.color, g.bgcolor, g.spot_size, g.size, g.projection = color, bgcolor, spot_size, size, projection
     return color, bgcolor, spot_size, size, projection
```

Comment:
Do we need this line?

Reply:
If you mean the ` g.color, g.bgcolor, g.spot_size, g.size, g.projection = color, bgcolor, spot_size, si
ze, projection`, yes we need this because if we do not update the attributes here when we are saving config or data the old values of attributes will be saved.

And about the next line, we return these values to use them in caller function.
You mean we don't need to return anything so we'll work with attributes, right?

Reply:
> And about the next line, we return these values to use them in caller function. You mean we don't need to return anything so we'll work with attributes, right?
Yeah